### PR TITLE
HIVE-26477: `CREATE TABLE LIKE STORED BY ICEBERG` failing with NPE

### PR DESCRIPTION
**Pull request by lcspinter**
### What changes were proposed in this pull request?
Throw a SemanticException when trying to create iceberg table using CTLT command.

### Why are the changes needed?
CTLT is not yet supported for iceberg tables. Until this is fixed we should warn the end user. 

### How was this patch tested?
Unit test

### CODE REVIEW

File: ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java [hunk 1]
```diff
@@ -13906,8 +13906,12 @@ ASTNode analyzeCreateTable(
       rootTasks.add(TaskFactory.get(new DDLWork(getInputs(), getOutputs(), crtTranTblDesc)));
       break;
 
-    case CTLT: // create table like <tbl_name>
+    case CTLT:// create table like <tbl_name>
 
+      if (handler != null && !handler.supportsCTLT()) {
+        throw new SemanticException("Unsupported operation. " + qualifiedTabName.getTable() +
+            " cannot be created using CTLT syntax. ");
```
Comment (ayushtkn) [inline ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java:13913]: Should we have an entry in the ErrorMsg.java for this case. There are couple of similar instanses for unspupported cases like ``TRUNCATE_FOR_NON_MANAGED_TABLE`` & for CTAS as well ``CTAS_LOCATION_NONEMPTY`` we can we have one like ``CTLT_FOR_ICEBERG_TABLE`` for this case?

Comment (lcspinter) [reply ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java:13913]: Good idea. I will add a new error type to the class.

**Review by szlta — APPROVED** [review]
+1

**Comment by lcspinter** [comment]
CTLT support was fixed in #3577. Abandoning this change.

**Pull request closed without merging**